src/platform/silabs/wifi/wf200/platform/wf200_init.c

```diff
@@ -319,6 +319,7 @@ sl_status_t sl_wfx_host_reset_chip(void)
  *****************************************************************************/
 sl_status_t sl_wfx_host_wait_for_wake_up(void)
 {
+    xSemaphoreTake(wfx_wakeup_sem, pdMS_TO_TICKS(0));
```

Comment:
why was this added ?

Reply:
I had initially removed this because it seemed like an odd thing to do. After double checking the Si SDK file from which this was created from, that is what the platform do so i added the line back to match.